java/Graql.java

```diff
@@ -247,15 +248,20 @@ private static Statement hiddenVar() {
     }
 
     @CheckReturnValue
-    public static StatementType type(Graql.Token.Type type) {
-        return type(type.toString());
+    public static StatementType type(Label label) {
+        return hiddenVar().type(label);
```

Comment:
Why are we replacing `.type(Graql.Token.Type type)` with `.type(Label label)`, Josh? This was supposed to be used strictly for "native" types, right? Also, there are already 2 methods taking in labels as Strings below; they should be enough, right? Do we need one that strictly takes in an argument of `Class<Label>`? The user will then have to manually construct a `Label` class instance, to pass in, right?